thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -167,39 +172,40 @@
       {{!-- Alert anomaly table --}}
       <div class="te-block-container">
         {{#if baselineOptionsLoading}}
-          <div class="spinner-wrapper spinner-wrapper--card">{{ember-spinner}}</div>
+          <div class="spinner-wrapper-self-serve spinner-wrapper-self-serve__custom">{{ember-spinner}}</div>
```

Comment:
Why don't we augment `spinner-wrapper` instead of overriding it? 
we could just add a modifier like so `class="spinner-wrapper spinner-wrapper--self-serve"`

So that we keep the same base class, and `modify` with `--modifier` classes

Reply:
yes, I like that better.